examples/7_qst_circuit.jl

```diff
@@ -1,6 +1,8 @@
 using PastaQ
 using Random
 using ITensors
+using Observers
+import Optimisers
```

Comment:
I prefer `using Optimisers: Optimisers`. It's the same thing as `import Optimisers` but doesn't let you overload a function like `Descent` without importing it explicitly or using `Optimisers.Descent`.

Reply:
Do you mean to use this in `PastaQ.jl` file? I'm not sure why it is that important in an example file

Reply:
Ah yeah I didn't notice this was an example. Not a big deal, but even for an example I think it is a better style since a user is less likely to accidentally overwrite a function in another package with `using` vs. `import`.

Though in this case can't we just do `using Optimisers` and then using `Descent` directly?

Reply:
Yes. Would you like to use this in the PastaQ.jl as well?

Reply:
Yeah that seems better.